ConversionTool.Test/ConversionToolTest.cs

```diff
@@ -0,0 +1,67 @@
+using ConversionTool.Controllers;
+using ConversionTool.HtmlToImage;
+using ConversionTool.Model;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using System.Drawing;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ConversionTool
+{
+    public class ConversionToolTest
+    {
+        private const string TEST_FOLDER = "TestFiles";
+        private const string HTML_FILE_1 = "UserPaymentConfirmTemplate.html";
+        private const string HTML_FILE_1_EXPECTED_CONTENT = "UserPaymentConfirmTemplateExpectedBase64.txt";
+
+        [Fact]
+        public async Task HtmlToImageController_Should_return_a_valid_png_image()
+        {
+            // Arrange
+            var basePath = GetPathToUnitTestProjectFiles();
+
+            var htmlFile = Path.Combine(basePath, TEST_FOLDER, HTML_FILE_1);
+            var htmlAsBytesFile = Path.Combine(basePath, TEST_FOLDER, HTML_FILE_1_EXPECTED_CONTENT);
+
+            var configPath = Path.Combine(basePath, "appsettings.json");
+
+            using var readerHtmlFile = new StreamReader(htmlFile);
+            var expectedResponseAsBytes = File.ReadAllBytes(htmlAsBytesFile);
+
+            var htmlText = await readerHtmlFile.ReadToEndAsync();
+
+            var htmlToImageMock = new Mock<IHtmlToImage>();
+            htmlToImageMock.Setup(x => x.FromStringToPngAsync(htmlText))
+                .ReturnsAsync(expectedResponseAsBytes);
+
+            var htmlToImageController = new HtmlToImageController(
```

Comment:
What do you think about this rename to indicate what is the _system under test_?
```suggestion
            var sut= new HtmlToImageController(
```